File: Persegi/C++/persegi-fill-i.cpp
#include <iostream>
using namespace std;

int main(){
    
    int x;
    cout << "Masukan Lebar Persegi : ";
    cin >> x;
    for(int i = 0; i <= x; i++){
        for(int j = 0; j <= x; j++){
            cout << "* ";
        }
        cout << endl;
    }

    return 0;
}

File: Persegi/C#/persegi-fill-i.cs
using System;
class fillPersegi {
  static void Main() {
    Console.Write("Masukan Lebar Persegi : ");
    int x = int.Parse(Console.ReadLine());
    for(int i = 0; i <= x; i++){
        for(int j = 0; j <= x; j++){
            Console.Write("* ");
        }
        Console.Write("\n");
    }
  }
}
